File: 771/771.py
# @brief:  leetcode-771-宝石与石头
# @envir:  python 2.7
# @date:   2019/5/5
# @author: luhao

class Solution(object):
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        return sum(S.count(i) for i in J) 

File: 771/771.cpp
/*!***********************
 * @brief:  leetcode-771-宝石与石头
 * @date:   2019/5/1
 * @envir:  gcc 7.4.0
 * @author: luhao
 * ***********************/

class Solution {
	public:
		int numJewelsInStones(string J, string S) {
			int ans = 0;
			for (int i=0; i<J.length(); i++){
				for (int j=0; j<S.length(); j++){
					if (J[i]==S[j])	
						ans ++;	
				}
			}
			return ans;             
		}
};

